astropy/io/registry.py

```diff
@@ -15,7 +17,7 @@
 
 __all__ = ['register_reader', 'register_writer', 'register_identifier',
            'identify_format', 'get_reader', 'get_writer', 'read', 'write',
-           'get_formats', 'IORegistryError']
+           'get_formats', 'IORegistryError', 'delay_doc_updates']
```

Comment:
Should this be exported at all? We only use it directly as `registry.delay_doc_updates`?